backend/src/processStream.ts

```
@@ -223,36 +229,45 @@ export const processCsv =  async (job: Job<any>, jobFile: JobInput): Promise<any
-        const scoredResults = scoreResults(chunk[idx], item.hits.hits.map(buildResultSingle), {...params})
-        if (scoredResults && scoredResults.length > 0) {
-          const selectedCanditates = scoredResults.slice(0, candidateNumber)
-          return selectedCanditates.map((selectedCanditate: any) => {
-            return {...chunk[idx], ...selectedCanditate}
-          })
+    return [{index: "deces"}, buildRequest(requestInput)];
+  }).flat()}
+
+  try {
+    const result =  await timerRunBulkRequest(bulkRequest);
+    if (result && result.responses && result.responses.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)